src/App.js

```diff
@@ -0,0 +1,100 @@
+import { Lightning, Utils } from 'wpe-lightning-sdk'
+import Menu from './mainMenu/menu.js'
+import OnDemand from './OnDemand/ondemand.js'
+
+export default class App extends Lightning.Component {
+  static getFonts() {
+    return [{ family: 'pixel', url: Utils.asset('../fonts/DejaVuSerif.ttf'), descriptor: {} }]
```

Comment:
As agreed please remove the DejaVuSerif.ttf font binary blob.

Reply:
font removed from getFonts method.